src/common/loggingEvents.ts

```diff
@@ -0,0 +1,71 @@
+import { EventType } from './EventType';
+import { PlatformInformation } from './PlatformInformation';
+import { Environment } from '../shared';
+
+export interface BaseEvent {

```

Comment:
can this be a concrete class and only use this one? And the user just set different `EventType` for it

Reply:
different event has different property, and `type` is just common property,

---
In reply to: [362393748](https://github.com/docascode/vscode-docs-build/pull/9#discussion_r362393748) [](ancestors = 362393748)